Cloud VM Exposure Remediation Playbooks (#44201)
demisto/content: Packs/CortexExposureManagement/ReleaseNotes/1_0_3.md

#### Triggers Recommendations

##### Cortex Exposure Management Enrichment

> - Updated the Cortex Exposure Management Enrichment triggers recommendation to omit issues with specific ASM rule IDs that are now handled in other triggers.

Reviewer: ```suggestion
Updated **Cortex Exposure Management Enrichment** triggers recommendation to omit issues with specific ASM rule IDs that are now handled in other triggers.
```

##### Cortex Exposure Management GCP Virtual Machine Remediation

> - Updated triggers recommendation - Cortex Exposure Management GCP Virtual Machine Remediation. This automation rule is responsible for handling remediation of Cortex Exposure Management issues for Virtual Machines in GCP, and replaces an issue filter that previously handled only RDP Server exposures.

Reviewer: ```suggestion
- Updated the ***triggers recommendation - Cortex Exposure Management GCP Virtual Machine Remediation*** automation rule to remediate Cortex Exposure Management issues for VMs in GCP. This replaces an issue filter that previously handled only RDP server exposures.
```

##### Cortex Exposure Management AWS Virtual Machine Remediation

> - Updated triggers recommendation - Cortex Exposure Management AWS Virtual Machine Remediation. This automation rule is responsible for handling remediation of Cortex Exposure Management issues for Virtual Machines in AWS EC2, and replaces an issue filter that previously handled only SSH Server exposures.

Reviewer: ```suggestion
Updated the ***triggers recommendation - Cortex Exposure Management AWS Virtual Machine Remediation*** automation rule to remediate Cortex Exposure Management issues for VMs in AWS EC2. This replaces an issue filter that previously handled only SSH server exposures.
```

##### Cortex Exposure Management Azure Virtual Machine Remediation

> - Added a new triggers recommendation - Cortex Exposure Management Azure Virtual Machine Remediation. This automation rule is responsible for handling remediation of Cortex Exposure Management issues for Virtual Machines in Azure.

Reviewer: ```suggestion
Added the ***triggers recommendation - Cortex Exposure Management Azure Virtual Machine Remediation*** automation rule to remediate Cortex Exposure Management issues for Virtual Machines in Azure.
```

#### Playbooks

##### New: Cortex EM - AWS VM Exposure Remediation

> This playbook handles exposure issue remediation for cloud hosted Virtual Machines by modifying cloud network security settings to block public access to the exposed service port.

Reviewer: ```suggestion
This playbook remediates public exposure for AWS hosted virtual machines by modifying network security settings to block public access to the exposed service port.
```

##### New: Cortex EM - Azure VM Exposure Remediation

> This playbook handles exposure issue remediation for cloud hosted Virtual Machines by modifying cloud network security settings to block public access to the exposed service port.

Reviewer: ```suggestion
This playbook remediates public exposure for Azure hosted virtual machines by modifying network security settings to block public access to the exposed service port.
```

##### New: Cortex EM - GCP VM Exposure Remediation

> This playbook handles exposure issue remediation for cloud hosted Virtual Machines by modifying cloud network security settings to block public access to the exposed service port.

Reviewer: ```suggestion
This playbook remediates public exposure for GCP hosted virtual machines by modifying network security settings to block public access to the exposed service port.
```

##### Cortex EM - Remediation

> - Deprecated. Use Cortex EM - AWS VM Exposure Remediation instead.

Reviewer: ```suggestion
Deprecated the **Cortex EM - Remediation** playbook. Use the **Cortex EM - AWS VM Exposure Remediation** playbook instead.
```
